docs/deployment/FPC_on_Azure.md

```diff
@@ -0,0 +1,139 @@
+# How to use an Azure Confidential Computing Instance to run FPC in HW mode
+Date of writing 14.03.2022.
+This guide is based on the following sources.
+[Koshi Ikegawa](https://qiita-com.translate.goog/ikegawa-koshi/items/8cf1fef1004fc16beb15?_x_tr_sl=ja&_x_tr_tl=en&_x_tr_hl=de&_x_tr_pto=wapp#fpc%E3%81%AE%E3%83%93%E3%83%AB%E3%83%89)
+
+- [How to use an Azure Confidential Computing Instance to run FPC in HW mode](#how-to-use-an-azure-confidential-computing-instance-to-run-fpc-in-hw-mode)
+  - [Creating the Confidential Computing Instance on Azure.](#creating-the-confidential-computing-instance-on-azure)
+  - [Prerequisites](#prerequisites)
+  - [Registering for the SGX Attestation Service Utilizing EPID](#registering-for-the-sgx-attestation-service-utilizing-epid)
+    - [Known Issues](#known-issues)
+  - [Building the FPC dev container](#building-the-fpc-dev-container)
+  - [Final preparation of the dev-container](#final-preparation-of-the-dev-container)
+
+## Creating the Confidential Computing Instance on Azure.
+Use the [Quick Create Portal](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/quick-create-portal) to create
+the virtual machine.
+Use the following parameters:
+* OS = *Linux (ubuntu 20.04)*
+* Size = Standard DC2s v2 (2 vcpus, 8 GiB memory)
+
+## Prerequisites
+Connect to your instance using ssh and execute the following commands
+```bash
+#update machine
+sudo -i
+apt-get update
+apt-get upgrade
+apt install -y docker docker-compose golang make
```

Comment:
I am wondering why one needs to install docker-compose and go? Later in this guide you use the FPV dev container, which already comes with go and docker-compose.

Reply:
I guess I needed go to " go get " the repository. but that is not necessary anymore.

I will leave docker and make in there and check if anything is needed on a "fresh" fpc vm